includes/widgets/text-editor.php

```diff
@@ -110,7 +110,7 @@ protected function _register_controls() {
 			[
 				'label' => '',
 				'type' => Controls_Manager::WYSIWYG,
-				'default' => __( 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut elit tellus, luctus nec ullamcorper mattis, pulvinar dapibus leo.', 'elementor' ),
+				'default' => '<p>' . __( 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut elit tellus, luctus nec ullamcorper mattis, pulvinar dapibus leo.', 'elementor' ) . '</p>',
```

Comment:
We have to understand first what caused this change

Reply:
Because of this commit:
https://github.com/elementor/elementor/commit/0a82ef41a06fe5ff42296bbc1dd2a3c0670f61df

![image](https://user-images.githubusercontent.com/10234691/76212547-61067280-6211-11ea-94ae-835c73fe4039.png)

the `SetContent` of  tinyMCE, was adding the `<p`>.